src/ai/backend/manager/api/session.py

```diff
@@ -1465,20 +1466,13 @@ async def report_stats(root_ctx: RootContext, interval: float) -> None:
 
 
 async def stats_report_timer(root_ctx: RootContext):
-    while True:
-        try:
-            aiotools.create_timer(
-                functools.partial(report_stats, root_ctx), 1.0,
-            )
-        except asyncio.CancelledError:
-            break
-        except Exception:
-            await root_ctx.error_monitor.capture_exception()
-            log.exception('stats_report_timer: unexpected error')
-        try:
-            await asyncio.sleep(5)
-        except asyncio.CancelledError:
-            break
+    try:
+        aiotools.create_timer(
```

Comment:
Timer creation should be inside the app initialization steps.

Reply:
Is it better that delete stats_report_timer and execute create_timer like [this line L2102~2103](https://github.com/lablup/backend.ai-manager/blob/b2362d3871a7a594f89469be5f6a5cbe256613bc/src/ai/backend/manager/api/session.py#L2102-L2103) ?